src/website/app/Nav.js

```diff
@@ -17,7 +17,7 @@ import sections from './pages';
 type Props = {
   currentDemo?: string,
   demoRoutes: Array<DemoRoute>,
-  wide?: Boolean
+  wide?: boolean
```

Comment:
Nice catch!

I am getting console errors when running this branch locally though...

----
Warning: Failed prop type: Invalid prop `wide` of type `Boolean` supplied to `Nav`, expected instance of `Boolean`.

Warning: Failed prop type: The prop `pageMeta` is marked as required in `ComponentDoc`, but its value is `undefined`.

Reply:
Hmmm, I think I was on an out of date version...  I updated and these errors went away.